src/styles/blocks/card.scss

```diff
@@ -0,0 +1,114 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 406px;
+  border: 1px solid $white-smoke;
+  border-radius: 5px;
+  margin-right: 48px;
+  margin-bottom: 46px;
+
+  &__img {
+    margin: 32px auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__inform {
+    margin: 0 16px;
+  }
+
+  &__product-name {
+    line-height: 18px;
+    font-weight: 500;
+    font-size: 12px;
+    color: $dark;
+    margin-top: 0;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    color: $gray;
+    font-size: 10px;
+    font-weight: 400;
+    margin-top: 0;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    @include with-centered-content;
+    margin-bottom: 24px;
+  }
+
+  &__title {
+    font-size: 10px;
+    @include margin-color;
+  }
+
+  &__price {
+    @include with-centered-content;
+    margin-bottom: 16px;
+
+    .card__price-title {
+      margin: 0;
+      color: $gray;
+      font-size: 12px;
+    }
+
+    .card__price-amount {
+      @include margin-color;
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
+
+  .stars {
+    display: flex;
+
+    .stars__star {
+      width: 16px;
+      height: 16px;
+      background-image: url("../images/star.svg");
+      background-position: center;
+      background-repeat: no-repeat;
+    }
+  }
+
+  .stars__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  .stars--1 .stars__star:nth-child(1),
+  .stars--2 .stars__star:nth-child(-n+2),
+  .stars--3 .stars__star:nth-child(-n+3),
+  .stars--4 :nth-child(-n+4),
+  .stars--5 .stars__star:nth-child(-n+5) {
+    background-image: url("../images/star-active.svg");
+  }
+
+  &__button {
+    padding: 0;
+    border: none;
+    font-family: inherit;
+    text-align: center;
+    justify-content: center;
+    width: 166px;
+    height: 40px;
+    background-color: $cerulean;
+    border-radius: 5px;
+    color: $white;
+    font-weight: 700;
+    font-size: 14px;
+  }
+
+  .card__button:hover {
```

Comment:
Nested classes should use the '&' symbol to maintain the BEM structure. Change the selector to '&:hover' instead of '.card__button:hover'.